somaticseq.wdl

```diff
@@ -0,0 +1,289 @@
+version 1.0
+
+import "common.wdl" as common
+
+task SomaticSeqParallelPaired {
+    input {
+        String? preCommand
```

Comment:
I am of the opinion we should drop this when we use containers. It hinders reproducibility greatly and has now added value anymore. (We hardly use it now as it is).